base/doc/clsguide.tex

```diff
@@ -977,6 +977,30 @@ \subsection{Passing options around}
 handler, but this handler is not used for |landscape| because that option is
 explicitly declared.
 
+\subsection{Testing dates of other code}
+
+\begin{decl}
+  |\IfPackageAtLeast(TF)| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfClassAtLeast(TF)| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfFileAtLeast(TF)| \arg{file} \arg{date} \arg{true code} \arg{false code}
+\end{decl}
+There are occasions when you need to test if one package if another package is
```

Comment:
Typo mentioned by @car222222 is still present:
> when you need to test if one package if another package is

And suggested was
> when you need to test *in* one package if another package is

But I think, this does not hold only for packages so should be removed completely

> when you need to test if another package is

Reply:
Changed